ckanext/openafrica/plugin.py

```diff
@@ -23,15 +23,16 @@
 
 import ckan.plugins as plugins
 import ckan.plugins.toolkit as toolkit
+from . import blueprint
 
 
 class OpenAfricaPlugin(plugins.SingletonPlugin):
     u"""
     openAFRICA templating plugin.
     """
-    plugins.implements(plugins.IConfigurer)
-    plugins.implements(plugins.IRoutes, inherit=True)
-    plugins.implements(plugins.ITemplateHelpers)
+    plugins.implements(plugins.IConfigurer, inherit=True)
+    plugins.implements(plugins.ITemplateHelpers, inherit=True)
+    plugins.implements(plugins.IBlueprint)
```

Comment:
Why do some plugins use inherit and others don't? (This is for just my understanding).

Reply:
On the blueprint, we only need custom routes to handle our `/about/<>/` urls hence though it was not necessary to extend CKAN core's Blueprints. On the other plugins, we inherit helper functions & configurations  such as `url_for`, `link_for` et.al from CKAN core